tmt/schemas/test.yaml

```diff
@@ -105,6 +105,16 @@ properties:
   test:
     type: string
 
+  setup:
+    anyOf:
+      - type: string
+      - type: array
+
+  cleanup:
+    anyOf:
+      - type: string
+      - type: array
```

Comment:
`array` should probably specify what items are allowed:

```yaml
- type: array
  items:
    type: string
```

Reply:
Fixed